.github/ISSUE_TEMPLATE/suggestion.yml

```diff
@@ -0,0 +1,44 @@
+name: Word Suggestion
+description: Share a word suggestion
+title: ":blue_book: "
+labels: ["Type · Definitions", "Status · Unconfirmed"]
```

Comment:
Language should be included somewhere in here

Reply:
These are the labels that get added automatically on creation. For the specific language, we would need to add some automation, either through GH actions or something else. We're going to capture language in the form, so maintainers can add the label after issue creation.